src/main/java/io/swagger/codegen/v3/generators/java/AbstractJavaCodegen.java

```diff
@@ -902,6 +911,11 @@ public CodegenModel fromModel(String name, Schema schema, Map<String, Schema> al
                 codegenModel.imports.add("Schema");
             }
         }
+
+        if(this.isNotNullJacksonAnnotation()){
+            codegenModel.imports.add("JsonInclude");
+        }
+
         if (codegenModel.discriminator != null && additionalProperties.containsKey("jackson")) {
```

Comment:
will this work also for cases where jackson is not used, e.g. added to POM?

Reply:
yes. I tested locally: generated and build with JDK 1.8 and worked!

Reply:
added to the pom in case the option is true